tagstudio/src/qt/widgets/__init__.py

```diff
@@ -10,4 +10,5 @@
 from .tag_box import TagBoxWidget
 from .text import TextWidget
 from .item_thumb import ItemThumb
-from .preview_panel import PreviewPanel
\ No newline at end of file
+from .preview_panel import PreviewPanel
+from .video_player import VideoPlayer
```

Comment:
All good here

Reply:
I didn't think about the import order, if preview panel relies on video player, then the video player needs to be before the preview panel in the __init__ file
```suggestion
from .video_player import VideoPlayer
from .preview_panel import PreviewPanel
```